src/lasp_core.erl

```diff
@@ -361,7 +361,7 @@ update(Id, Operation, Actor, Store) ->
     {ok, var()} | not_found().
 update(Id, Operation, Actor, MetadataFun, Store) ->
     {ok, #dv{value=Value0, type=Type}} = do(get, [Store, Id]),
-    {ok, Value} = lasp_type:update(Type, Operation, Actor, Value0),
+    {ok, Value} = lasp_type:update(Type, Operation, {Id, Actor}, Value0),
```

Comment:
Globally unique for an actor modifying an object or globally unique representing a globally unique event?